src/main/java/vertex/pro/edu/soung_box_app/SoundBoxApp.java

```diff
@@ -13,26 +13,26 @@
 @Slf4j
 @RequiredArgsConstructor
 @SpringBootApplication
-public class SoundBoxApp implements CommandLineRunner {
+public class SoundBoxApp {
 
-    private final SongRepository repository;
+//    private final SongRepository repository;
 
     public static void main(String[] args) {
         SpringApplication.run(SoundBoxApp.class, args);
     }
 
-    @Override
-    public void run(String... args) throws Exception {
-        for (int i = 0; i < 10; i++) {
-            SongEntity entity = SongEntity.builder()
-                    .album("Album")
-                    .artist("MONATIK")
-                    .genre(ThreadLocalRandom.current().nextBoolean() ? "Rock" : "R&B")
-                    .title("Title")
-                    .build();
-
-            log.info("Entity: {}", entity);
-            repository.save(entity);
-        }
-    }
+//    @Override
```

Comment:
Please delete commented code

Reply:
But these commented code maybe needed for other people to fill their repository with songs